### PR TITLE
INFRA-4977 zombie counter INFRA-4978 docker_container_count

### PR DESCRIPTION
**Pull request by ahammond**

### CODE REVIEW

**Review by sund0g — COMMENTED** [review]
Approved by me but would like additional review by @onorua and/or @white-hat

File: main.go [hunk 1]
```diff
@@ -188,20 +201,147 @@ func scrapeDockerVersion(cli *client.Client) {
...
+}
+
+func (p *UnixProcess) Refresh() error {
+	statPath := fmt.Sprintf("/proc/%d/stat", p.pid)
+	dataBytes, err := ioutil.ReadFile(statPath)
+	if err != nil {
+		return err
+	}
+
+	// First, parse out the image name
```
Comment (white-hat) [inline main.go:248]: please add a sample string

File: main.go [hunk 2]
```diff
@@ -188,20 +201,147 @@ func scrapeDockerVersion(cli *client.Client) {
...
+			}
+
+			// Look for executables with names like [docker-stop] <defunct>
+			if strings.HasPrefix(p.binary, "[docker") {
+				zombieCount++
+			}
+		}
+	}
+
+	d.Close()
```
Comment (white-hat) [inline main.go]: already closed at #263

**Review by white-hat — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (ahammond) [reply main.go:248]: Added.

**Review by white-hat — APPROVED** [review]

**Pull request merged**